benchmark.py

```diff
@@ -235,8 +256,16 @@ def _run_benchmark(bazel_binary_path,
     bazelrc: the path to a .bazelrc file.
     prefetch_ext_deps: whether to do a first non-benchmarked run to fetch the
       external dependencies.
+    bazel_bench_uid: a unique string identifier of this entire bazel-bench run.
     bep_json_dir: absolute path to the directory to write the build event json
       file to.
+    collect_json_profile: whether to collect JSON profile for each run.
+    data_directory: the path to the directory to store run data.
+      Required if collect_json_profile.
```

Comment:
you have asserts below for all these requirements. I would prefer nice human readable error messages.

Reply:
Done